pom.xml

```diff
@@ -12,10 +12,10 @@
     <description>project-description</description>
     <properties>
         <!-- java -->
-        <java.version>11</java.version>
+        <java.version>16</java.version>
```

Comment:
Es wäre vielleicht besser, wenn wir Java 17 nutzen würden.
Das ist die neuste LTS Version.

Reply:
I will update it tomorrow.